ntest: don't ignore unknown console.error's

jelly opened the pull request:
Our console.error verification code did not work as expected as our default_allowed_console_errors list contained "" which can always be found in log messages.

jelly commented:
```
error: Tracer failed: "{"problem":"access-denied","exit_status":null,"exit_signal":null,"message":"Not permitted to perform this action."}", data: "undefined"
```

This needs to be allowed.

jelly commented:
We are left with:

* [ ] TestSuperuser.testMultipleBridgeConfig `error: FormSelect requires either an id or aria-label to be specified`
* [ ] TestApps.testWithUrlRoot ```testlib.Error: FAIL: Test completed, but found unexpected browser errors:
error: 

# Result testWithUrlRoot (__main__.TestApps.testWithUrlRoot) failed
```
* [ ] TestStorageLuks.testLuks/testNoFs `error: Text input: Text input requires either an id or aria-label to be specified`

jelly commented:
```
> error: Text input: Text input requires either an id or aria-label to be specified
> warning: transport closed: disconnected
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/common/testlib.py", line 1598, in tearDown
    self.check_browser_errors()
  File "/work/bots/make-checkout-workdir/test/common/testlib.py", line 1868, in check_browser_errors
    raise Error(UNEXPECTED_MESSAGE + "browser errors:\n" + log)
testlib.Error: FAIL: Test completed, but found unexpected browser errors:
error: Text input: Text input requires either an id or aria-label to be specified

# Result testCreate (__main__.TestTimers.testCreate) failed
```


```
# testBasic (__main__.TestApps.testBasic)
Failed to kill unit packagekit.service: Unit packagekit.service not loaded.
Failed to reset failed state of unit packagekit.service: Unit packagekit.service not loaded.
rm: cannot remove '/etc/systemd/system/dnf-automatic*': No such file or directory

DevTools listening on ws://127.0.0.1:9487/devtools/browser/45c78bab-0571-4b02-aadc-fe8dd6e9d2cc
[0705/114929.437608:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
warning: source_date_epoch_from_changelog set but %changelog is missing
warning: source_date_epoch_from_changelog set but %changelog is missing
> log: PackageKit went away from D-Bus
> log: PackageKit went away from D-Bus
> warning: PackageKit went away during transaction /9_ceabddcd : {"command":"close","channel":"1:5!5"}
> error: 
> warning: transport closed: disconnected
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/common/testlib.py", line 1598, in tearDown
    self.check_browser_errors()
  File "/work/bots/make-checkout-workdir/test/common/testlib.py", line 1868, in check_browser_errors
    raise Error(UNEXPECTED_MESSAGE + "browser errors:\n" + log)
testlib.Error: FAIL: Test completed, but found unexpected browser errors:
error: 
```
```

martinpitt commented:
I locally rebased this PR. Many of the commits are already in main, and I sent PR #19496 for some more. After that, let's rebase this and review what's left. @jelly says Garrett has advised against adding unnecessary aria-labels.

jelly commented:
I'd suggest we let this run with just fedora-38/devel after a rebase and see what the damage is

martinpitt commented:
I rebased this and cleaned this up. See description for the aria stuff, let's handle that separately. Doing an initial run, and then I'll just add the aria bits to allowed messages. Then we have a nice laundry list of cleanups, and avoid introducing new ones.

jelly commented:
> I rebased this and cleaned this up. See description for the aria stuff, let's handle that separately. Doing an initial run, and then I'll just add the aria bits to allowed messages. Then we have a nice laundry list of cleanups, and avoid introducing new ones.

Ack, I agree with leaving the aria stuff for a later cleanup.

martinpitt commented:
The [first run](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19053-20231017-122029-f903d9a6-fedora-38-devel/log.html#141) is quite a mouthful. I went through with a coarse-grained `grep | sort` and ignore them wholesale. I suggest to land this ASAP, and then we can parallelize the actual fixes.

martinpitt commented:
I also changed the approach for fixing the `""` handling. Your original patch did not work very well, and it also turned "search" into "match" semantics.

martinpitt reviewed (approved):
This is sort of weird.. Mostly my changes now, but @jelly 's PR. I formally approve, but @jelly please re-review and land if you are happy.

jelly reviewed (commented):
LGTM

martinpitt commented:
Argh, [one more](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19053-20231017-173628-dedc93e8-fedora-38-devel/log.html#131), due to a stupid typo. Fixed. It's blindingly obvious, so re-approving.
